bundles/org.openhab.ui/web/src/components/widgets/plan/oh-plan-page.vue

```diff
@@ -143,20 +150,34 @@ export default {
       zoom: -0.5,
       crs: CRS.Simple,
       showMap: true,
-      mapOptions: {
-        zoomSnap: 0.1
-      }
+      mapKey: this.$f7.utils.id()
     }
   },
   computed: {
     bounds () {
       const lat = this.config.imageHeight || 1000
       const lng = this.config.imageWidth || 1000
       return [[0, 0], [lat, lng]]
-    }
+    },
+    mapOptions () {
+    return Object.assign({
+      zoomSnap: 0.1
+    }, this.config.noZoomOrDrag ? {
+      dragging: false, 
+      touchZoom: false, 
+      doubleClickZoom: false,
+      scrollWheelZoom: false, 
+      zoomControl:false
+      } : {})
+    },
   },
   mounted () {
-    this.$refs.map.mapObject.fitBounds(this.bounds)
+    this.fitMapBounds()
+  },
+  watch: {
+    'config.noZoomOrDrag' : function (val) {
```

Comment:
As general advice,,watchers are normally discouraged - but here it makes sense in this case :)
Though we'd probably check in a debugger if it's called when the map is first rendered (even if the option isn't set), in which case we wouldn't need the `fitMapBounds()` call in `mounted`; the goal would be to call it only once if the config doesn't change.

Reply:
Just checked and it's only calling fitMapBounds when the config option changes